.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!kvm-operator
+!go.mod
```

Comment:
Why is a dockerignore needed?

Reply:
I'm actually not convinced that it is anymore -- removing to see what happens